control-plane/csi-driver/src/bin/node/nodeplugin_grpc.rs

```diff
@@ -54,6 +68,71 @@ impl NodePlugin for NodePluginSvc {
             device_path: device.devname(),
         }))
     }
+
+    #[tracing::instrument(err, skip_all)]
```

Comment:
Add volume id here, similar to csi node.rs?

Reply:
Done.

Reply:
`#[tracing::instrument(err, fields(volume.uuid = request.get_ref().volume_id), skip(self))]`